walkthroughs/writing-walkthroughs/walkthrough.adoc

```diff
@@ -453,3 +453,87 @@ Check that the step is displayed as a link to the Openshift Console.
 
 [type=verificationFail]
 Check that the correct attribute has been used.
+
+[time=10]
+== Adding custom services to a walkthrough
+
+As an advanced task, you can create services from the OpenShift catalog in the cluster. You can also link to the URLs associated with those services in your `walkthrough.adoc` file.
+
+=== Finding the name of the service you want to provision
+
+.Prerequisites
+
+Some experience with the `oc` command. See task resources for more information.
+
+
+. Log into the link:{openshift-url}[Openshift Console].
+
+. Click your username in the top right corner and choose *Copy Login Command* from the menu.
+
+. Log into the OpenShift CLI using the `oc` on your clipboard.
+
+. Run the following command to list all the service names available from your cluster catalog:
++
+----
+oc get clusterserviceclasses -o custom-columns="displayName:.spec.externalMetadata.displayName, externalName:.spec.externalName"
+----
++
+Typically, service names describe many variations of components. 
```

Comment:
@finp I think an extra line to explain what you see in the output of above command would be helpful.
There are 2 columns, but what are they?
Also, what column should the user be concerned with for the next step?

Reply:
This is a good point, I also noticed the column names blend in with the rest of the names. Maybe we could do something like?

```
oc get clusterserviceclasses -o custom-columns="Display Name (From Service Catalog):.spec.externalMetadata.displayName, Identifier (clusterServiceClassExternalName):.spec.externalName"
```

The table was created as a mapping between the names the user would know and understand from the Service Catalog UI (left-hand side) and the names/identifiers the user needs when specifying a `clusterServiceClassExternalName` in a ServiceInstance.

Reply:
This looks better.
Referencing the `clusterServiceClassExternalName` from the column for what value to use in  the next step helps